src/Symfony/Bundle/WebProfilerBundle/EventListener/ProfilerLinkLogListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @author Jérémy Romey jeremyFreeAgent <jeremy@free-agent.fr>
+ */
+final class ProfilerLinkLogListener implements EventSubscriberInterface
+{
+    public function __construct(
+        private ?LoggerInterface $logger = null,
+        private ?UrlGeneratorInterface $urlGenerator = null,
+    ) {
+    }
+
+    public function onKernelResponse(ResponseEvent $event): void
+    {
+        if (null === $this->logger) {
+            return;
+        }
+        if (null === $this->urlGenerator) {
+            return;
+        }
+
+        $response = $event->getResponse();
+        $request = $event->getRequest();
+
+        if (!$event->isMainRequest()) {
+            return;
+        }
+
+        if (!$response->headers->has('X-Debug-Token')) {
+            return;
+        }
+
+        $this->logger->debug(\sprintf('See profiler at %s', $this->urlGenerator->generate('_profiler', ['token' => $response->headers->get('X-Debug-Token')], UrlGeneratorInterface::ABSOLUTE_URL)));
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            KernelEvents::RESPONSE => ['onKernelResponse', -2048],
```

Comment:
perhaps unrelated, but does the console command terminate event can also be hooked here ?
so both web/cli have this log ?